src/cli/commands/cid.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const path = require('path')
+
+const cidCommandsPath = path.join(
+  __dirname, '..', '..', '..', 'node_modules', 'cid-tool', 'src', 'cli', 'commands'
+)
+
+module.exports = {
+  command: 'cid <command>',
+
+  description: 'Convert, format and discover properties of CIDs.',
+
+  builder (yargs) {
+    return yargs
+      .commandDir(cidCommandsPath)
```

Comment:
woa, this is magic. Running a CLI from another CLI ⚡️

Reply:
I got the idea from js-ipfs-mfs ;)